examples/pytorch/summarization/run_summarization.py

```diff
@@ -114,6 +120,9 @@ class DataTrainingArguments:
     Arguments pertaining to what data we are going to input our model for training and eval.
     """
 
+    source_lang: str = field(default=None, metadata={"help": "Source language id for summarization."})
+    target_lang: str = field(default=None, metadata={"help": "Target language id for summarization."})
```

Comment:
If the language is different for the inputs and the outputs it is a translation task, not a summarization task, so there should only be one language argument.